Project1/ExtendedCalculator.cs

```diff
@@ -8,5 +8,17 @@ namespace Project1
 {
     internal class ExtendedCalculator
     {
+        public bool PrimeNumber(int num1)
+        {
+            bool primenumber = true;
+            for (int i = 2; i < num1; i++)
+            {
+                
+                if (num1 % i == 0)
+                    primenumber=false;
```

Comment:
W ifie dla optymalizacji daj breaka. Tak że jak raz podzieli liczbę to po co ma dalej sprawdzać inne wyniki skoro już wiadomo że jest to liczba pierwsza